modules/task/backend.go

```diff
@@ -30,6 +30,12 @@ import (
 
 var gobEncoding = &GobEncoding{}
 
+const (
+	_initialized = iota
```

Comment:
Either one is ok but we should be consistent across our repo. Let me know if you want to start following no underscores for constants.